preset.py

```diff
@@ -0,0 +1,146 @@
+import config
+
+def place(board, shape, a, b):
+    """Places a preset shape."""
+    if shape == 1:
+        if a + 2 >= config.Width + config.Cushion:
+            a = config.Width + config.Cushion - 3
+        if b + 2 >= config.Height + config.Cushion:
+            b = config.Height + config.Cushion - 3
+        board[a][b].kill()
+        board[a + 1][b].birth(config.Square)
+        board[a + 2][b].kill()
+        board[a][b + 1].kill()
+        board[a + 1][b + 1].kill()
+        board[a + 2][b + 1].birth(config.Square)
```

Comment:
looks awful, i'd look into making the board a numpy array and using this https://stackoverflow.com/questions/26975769/modify-a-particular-row-column-of-a-numpy-array

Reply:
Not saying that it is possible, but i;d look into possible solutions

Reply:
Being able to modify a row/column won't help, most of the presets annoyingly don't follow any pattern at all so as far as I am aware this is the best way to place them.